Makefile

```
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.0
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.3
```

```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.4
```

bump to the newer version here and other places

---

already updated to 1.4 in the main branch